base/config/spinnaker.yml

```diff
@@ -17,9 +23,21 @@ services:
   echoWorker:
     baseUrl: http://echo-worker.spinnaker:8089
     enabled: true
+  fiat:
+    baseUrl: http://fiat.spinnaker:7003
+    enabled: true
   front50:
     baseUrl: http://front50.spinnaker:8080
     enabled: true
+  # TODO(ezimanyi): The gate URL might be http:// or https://. Figure out how
+  # to handle both cases. (Or, alternately, determine if we need this URL here
+  # at all given that other services should not be sending requests to gate.)
```

Comment:
Ooh yeah we might not need this; with Halyard, `gate.baseUrl` is written directly to Deck's profile so it doesn't read from this.